components/_patterns/01-atoms/lists/_lists.scss

```diff
@@ -0,0 +1,108 @@
+$color-list-bullet: $blue-dark;
```

Comment:
This isn't a color in our scheme is it? Maybe double-check this PR for ithaca specific pieces.